iroha_version/iroha_version_derive/src/lib.rs

```diff
@@ -62,18 +62,18 @@ pub fn version_with_json(attr: TokenStream, item: TokenStream) -> TokenStream {
 /// use iroha_version_derive::{declare_versioned, version};
 /// use iroha_version::json::*;
 ///
-/// declare_versioned!(VersionedMessage 1..2);
+/// declare_versioned!(VersionedValidatedMessage 1..2);
 ///
-/// #[version(n = 1, versioned = "VersionedMessage")]
+/// #[version(n = 1, versioned = "VersionedValidatedMessage")]
 /// #[derive(Debug, Clone, Decode, Encode, Serialize, Deserialize)]
-/// pub struct Message1;
+/// pub struct ValidatedMessage1;
 ///
-/// let versioned_message: VersionedMessage = Message1.into();
+/// let versioned_message: VersionedValidatedMessage = ValidatedMessage1.into();
 /// let json = versioned_message.to_versioned_json_str().unwrap();
-/// let decoded_message = VersionedMessage::from_versioned_json_str(&json).unwrap();
+/// let decoded_message = VersionedValidatedMessage::from_versioned_json_str(&json).unwrap();
 /// match decoded_message {
-///    VersionedMessage::V1(message) => {
-///        let _message: Message1 = message.into();
+///    VersionedValidatedMessage::V1(message) => {
+///        let _message: ValidatedMessage1 = message.into();
```

Comment:
Is it intentional? Seems like auto-replacement as these names have no meaning here.

Reply:
Sorry, will fix now